src/views/v2/tickets/item/content/title.php

```diff
@@ -48,6 +48,11 @@
 
 $no_description = ! $ticket->show_description() || empty( $ticket->description ) || $is_mini;
 
+$container_classes = [
```

Comment:
We'll need to set the `@since TBD changes description.` for this template. And the `@version TBD`.